09-features.md

```diff
@@ -43,6 +43,7 @@ The Context column decodes as follows:
 | 26/27 | `option_shutdown_anysegwit`         | Future segwit versions allowed in `shutdown`              | IN       |                   | [BOLT #2][bolt02-shutdown]   |
 | 44/45 | `option_channel_type`            | Node supports the `channel_type` field in open/accept     | IN       |                   | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 48/49 | `option_payment_metadata` | Payment metadata in tlv record | 9 | | [BOLT #11](11-payment-encoding.md#tagged-fields)
+| 52/53 | `option_htlc_hold`               | Hold HTLCs and forward on receipt of an onion message     | IN       | `option_onion_messages`                                   |
```

Comment:
I believe we do want to advertise this to everyone, because it's a good selling point for people to open channels with you: by just looking at your `node_announcement` they know that you offer this feature that will be useful to them if they want to receive payments and expect to be offline sometimes.